boards/feather-m0/include/periph_conf.h

```diff
@@ -177,9 +177,10 @@ static const adc_conf_chan_t adc_channels[] = {
     { GPIO_PIN(PA, 4), ADC_INPUTCTRL_MUXPOS_PIN4 },     /* A3 */
     { GPIO_PIN(PA, 5), ADC_INPUTCTRL_MUXPOS_PIN5 },     /* A4 */
     { GPIO_PIN(PB, 2), ADC_INPUTCTRL_MUXPOS_PIN10 },    /* A5 */
+    { GPIO_PIN(PA, 7), ADC_INPUTCTRL_MUXPOS_PIN7 },     /* A7 */
```

Comment:
Maybe also mention that this is connected to analog battery level or something ?

Reply:
Where? I thought the usual approach with boards is to just rely on the board's references if they exist. Since the Feather has an excellent [pinout diagram](https://cdn-learn.adafruit.com/assets/assets/000/046/196/original/Feather_M0_Basic_Proto_v2.2.pdf?1504805559) that even I can figure out at one glance, I'd say we are golden with this.

Reply:
> Where ?

My idea is to have it in the comment, after `A7`. But the best would be to add this to the board description in the Doxygen documentation.

Reply:
Done. I just noticed: in the pinout diagram on the doc page there is also a `ADC_INPUTCTRL_MUXPOS_PIN19` on PA11 note done. Maybe that's the hidden A6? So maybe something for a follow-up?

Reply:
In fact, A7 is not really hidden, it's mapped to D9 on the board pinout. On the pinout, I see PA11 connected to UART RX, are you sure ?

Otherwise, thanks for updating the documentation, it's much better this way :)

Reply:
I guess you mean A6. Otherwise, I am confused ^^". I know its UART RX, but that's what the graphic says. Maybe because of that it is not documented in newer graphics like the one I linked in OP.

Reply:
A7 is D9.

Reply:
(which is the one I map here)

Reply:
See https://github.com/RIOT-OS/RIOT/pull/10938#discussion_r259323370 why `ADC_LINE(6)` == A7. ;-)